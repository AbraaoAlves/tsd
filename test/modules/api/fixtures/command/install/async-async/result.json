{
  "nameMatches": [
    {
      "path": "async/async.d.ts",
      "project": "async",
      "name": "async",
      "def": {
        "path": "async/async.d.ts",
        "commit": {
          "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
        },
        "blob": {
          "sha": "5ba56e0dbb7cc433d5b98ab3cbbf5e0d92954ff1"
        },
        "key": "async/async.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
        "solved": true,
        "dependencies": []
      },
      "history": []
    }
  ],
  "selection": [
    {
      "path": "async/async.d.ts",
      "commit": {
        "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
      },
      "blob": {
        "sha": "5ba56e0dbb7cc433d5b98ab3cbbf5e0d92954ff1"
      },
      "key": "async/async.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "solved": true,
      "dependencies": []
    }
  ],
  "written": {
    "D:\\_Editing\\github\\tsd\\tsd-origin\\test\\modules\\api\\tmp\\command\\install\\async-async\\typings\\async\\async.d.ts": {
      "path": "async/async.d.ts",
      "commit": {
        "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
      },
      "blob": {
        "sha": "5ba56e0dbb7cc433d5b98ab3cbbf5e0d92954ff1"
      },
      "key": "async/async.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "solved": true,
      "dependencies": []
    }
  }
}