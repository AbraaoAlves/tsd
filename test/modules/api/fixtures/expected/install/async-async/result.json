{
  "nameMatches": [
    {
      "path": "async/async.d.ts",
      "project": "async",
      "name": "async",
      "def": {
        "path": "async/async.d.ts",
        "commit": {
          "commitSha": "3bc329f49a971b3dae92ca682cb81816dbe31a53"
        },
        "blob": {
          "sha": "5ba56e0dbb7cc433d5b98ab3cbbf5e0d92954ff1"
        },
        "key": "async/async.d.ts-3bc329f49a971b3dae92ca682cb81816dbe31a53",
        "solved": true,
        "dependencies": []
      },
      "history": []
    }
  ],
  "selection": [
    {
      "path": "async/async.d.ts",
      "commit": {
        "commitSha": "3bc329f49a971b3dae92ca682cb81816dbe31a53"
      },
      "blob": {
        "sha": "5ba56e0dbb7cc433d5b98ab3cbbf5e0d92954ff1"
      },
      "key": "async/async.d.ts-3bc329f49a971b3dae92ca682cb81816dbe31a53",
      "solved": true,
      "dependencies": []
    }
  ],
  "written": {
    "D:\\_Editing\\github\\tsd\\tsd-origin\\test\\modules\\api\\tmp\\result\\install\\async-async\\typings\\async\\async.d.ts": {
      "path": "async/async.d.ts",
      "commit": {
        "commitSha": "3bc329f49a971b3dae92ca682cb81816dbe31a53"
      },
      "blob": {
        "sha": "5ba56e0dbb7cc433d5b98ab3cbbf5e0d92954ff1"
      },
      "key": "async/async.d.ts-3bc329f49a971b3dae92ca682cb81816dbe31a53",
      "solved": true,
      "dependencies": []
    }
  }
}